src/action.ts

```diff
@@ -18,13 +18,11 @@ export async function run() {
   const context = github.context;
   try {
     const assembly = AssemblyManifestReader.fromPath(inputs.cdkOutDir);
-    let stages = assembly.stages;
-    if (!stages.length) {
-      stages = [{
-        name: 'DefaultStage',
-        stacks: assembly.stacks,
-      }];
-    }
+    const stages = assembly.stages;
+    stages.push({
+      name: 'DefaultStage',
+      stacks: assembly.stacks,
+    });
```

Comment:
```suggestion
    if (assembly.stacks.length) {
      stages.push({
        name: 'DefaultStage',
        stacks: assembly.stacks,
      });
    }
```

Just to be safe can we only add the default stage if we have stacks?

Reply:
Makes sense to me